### PR TITLE
Make some style 

### PR DESCRIPTION
**Pull request by vladfreel**
Make some style for index, show, edit and new views

### CODE REVIEW

File: app/views/books/show.html.haml [hunk 1]
```diff
@@ -55,7 +55,10 @@
           %strong= @book.name
         %p
           Статус:
-          %strong= @book.status
+          - if @book.status == 'in'
+            %img{src: "/uploads/green2.png", class: 'status'}
+          - else
+            %img{src: "/uploads/red.png", class: 'status'}
```
Comment (drkmen) [inline app/views/books/show.html.haml]: %img{src: @book.status == 'in' ? "/uploads/green2.png" : "/uploads/red.png", class: 'status'}

File: app/views/books/_form.html.haml [hunk 2]
```diff
@@ -1,36 +1,47 @@
...
-    %img{src: '/uploads/book.png', class: 'logo'}
-  %div
-    %p
-      Search
-      %input{type: 'search', class: 'search'}
+%body.body_color
+  .back_row
+    .row
+    .col-lg-2
+
```
Comment (drkmen) [inline app/views/books/_form.html.haml:5]: avoid empty blocks

**Review by drkmen — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -4,10 +4,92 @@
   body{
     text-align: center;
   }
+  .body_color{
+    background-color: darkkhaki;
+  }
```
Comment (drkmen) [inline app/assets/stylesheets/application.scss]: use empty line between each block { }
so it will be:
.div {
 ...
}

.span {}

Comment (capitalsav) [inline app/assets/stylesheets/application.scss]: useless empty line in 74 and 81 lines

**Pull request merged**